packages/firestore/src/remote/serializer.ts

```diff
@@ -595,11 +595,11 @@ export class JsonProtoSerializer {
       !!doc.found,
       'Tried to deserialize a found document from a missing document.'
     );
-    assertPresent(doc.found!.name, 'doc.found.name');
-    assertPresent(doc.found!.updateTime, 'doc.found.updateTime');
-    const key = this.fromName(doc.found!.name!);
-    const version = this.fromVersion(doc.found!.updateTime!);
-    return new Document(key, version, {}, undefined, doc.found!, v =>
+    assertPresent(doc.found.name, 'doc.found.name');
+    assertPresent(doc.found.updateTime, 'doc.found.updateTime');
```

Comment:
Feel free to punt, but could we magic up assertPresent() to get rid of the remaining `!` usage below?

Reply:
Updated. Do you want to take another look?